packages/docs/docs/contributing/bounty.md

```diff
@@ -0,0 +1,35 @@
+---
+image: /generated/articles-docs-contributing-bounty.png
+title: How to take a bounty issue
```

Comment:
Consider writing it without "How to" - every title of documentation could technically start with "How to" so it's kind of a filler word for us.